include/secure_services.h

```diff
@@ -58,18 +58,13 @@ void spm_request_system_reboot(void);
 
 /** Request a random number from the Secure Firmware.
  *
- * This provides a True Random Number from the on-board random number generator.
+ * This provides a CTR_DRBG number from the CC3XX platform libraries.
  *
```

Comment:
Nit: Mention that CTR_DRBG means pseudorandom

Reply:
Will be changed to PRNG since we will have the option to use both HMAC_DRBG and CTR_DRBG (soon)